src/index.spec.ts

```diff
@@ -1,12 +1,227 @@
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+enableFetchMocks();
```

Comment:
I would have preferred to mock the `node-fetch` function that is the one used in the index.js file